.pre-commit-config.yaml

```diff
@@ -61,6 +57,7 @@ repos:
     rev: '2.2.157'
     hooks:
       - id: checkov
-        args: ['--framework', 'cloudformation', '--quiet']
-        files: ^(?!\..*).*/template\.(yml|yaml)$
-# - repo: https://github.com/mattlqx/pre-commit-sign
+        verbose: true
```

Comment:
is there a change needed in this file ?

Reply:
@maruf-ahmedd can check these changes if required?